mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerLocalScan.java

```diff
@@ -311,6 +313,26 @@ public void testCreateTableWithColumnSpecificationMultilevelPartitioned() throws
     runCreateAndReadTest(identifier, createSql, HiveIcebergStorageHandlerTestUtils.CUSTOMER_SCHEMA, spec, data);
   }
 
+  @Test
+  public void testExternalTableToTableManagedInHiveCatalog() throws IOException {
+    Assume.assumeTrue("Only relevant for HiveCatalog", testTableType == TestTables.TestTableType.HIVE_CATALOG);
+
+    TableIdentifier identifier = TableIdentifier.of("default", "customers");
+    testTables.createIcebergTable(shell.getHiveConf(), identifier.name(),
+        HiveIcebergStorageHandlerTestUtils.CUSTOMER_SCHEMA, FileFormat.PARQUET,
+        Collections.emptyList());
+
+    Map<StructLike, List<Record>> data = Maps.newHashMap();
+    data.put(null, HiveIcebergStorageHandlerTestUtils.CUSTOMER_RECORDS);
+
+    String createSql = String.format("CREATE EXTERNAL TABLE external_table_to_iceberg " +
+        "STORED BY 'org.apache.iceberg.mr.hive.HiveIcebergStorageHandler' " +
+        "TBLPROPERTIES('name'='%s')", identifier);
```

Comment:
The `name` property seems like to be an internal property for convertion between hive table and iceberg table, I'm not sure it is correct to expose as a table property like this to hive sql users.

Reply:
I think while using `name` is the most convenient, using something like `alias_to_table=xxx` or `referenced_table=xxx` would be more descriptive to users and would avoid leaking the internal `name` property. That would require making code changes in StorageHandler, SerDe, etc. though. What do you think?

Reply:
Agree,  Then I think we add document in the hive section to describe/explain the property `name` for end users.